libs/comparator/tables.js

```diff
@@ -39,6 +39,13 @@ const buildColumnsMap = table => {
                 column.unique && (options.push('UQ'))
                 column.required && (options.push('NN'))
 
+                if (relation.metaInfo?.relationIdentificationColumnId) {
+                    column.metaInfo || (column.metaInfo = {})
+                    column.metaInfo.relationIdentificationColumnId = relation.metaInfo.relationIdentificationColumnId
+
+                    options.push('relationIdentificationColumnId')
```

Comment:
You are just pushing the same word for comparison instead of specifying the column **name**. 
the comparison must be made by name, not id. The differences should be printed in human-readable format, and only patching the schema should be done using the column id